lua/nvim-tree/actions/fs/copy-paste.lua

```diff
@@ -94,11 +94,11 @@ local function do_single_paste(source, dest, action_type, action_fn)
   end
 
   if dest_stats then
-    vim.ui.input({ prompt = dest .. " already exists. Overwrite? y/n/r(ename): " }, function(choice)
+    vim.ui.select({ "Yes", "No", "Rename" }, { prompt = dest .. " already exists. Overwrite? y/n/r(ename): " }, function(choice)
```

Comment:
This works when using dessing.

However, the user experience for vanilla vim is unacceptable:
```
/tmp/1499/aaa already exists. Overwrite? y/n/r(ename):
1: Yes
2: No
3: Rename
Type number and <Enter> or click with the mouse (q or empty cancels):
```

Reply:
so without dressing, the `input` is a disaster.
Both options are bad for some and good for others. can we let users decide?

Reply:
That is a reasonable idea. A single interaction that covers both cases does not seem feasible.

Add an option something like `select_popup` and provide alternate vim.ui.input/select for each of the invocations.

Before we add an option, is there a mechanism by which we can detect whether the user is using dressing of some sort? I believe there are several out there...

Reply:
I believe something like this could work:

```
local hasdressing, _ = pcall(require,"dressing")
if hasdressing then
  ...
end
```

Reply:
> pcall(require,"dressing")

This will work for dressing, however this is vim and so there will be many other vim.ui implementations. Adding a condition for each of these is not feasible.

Some digging has indicated that dressing and others function by overriding `vim.ui.select` and it does not appear possible to determine whether this override is in effect.

Therefore two implementations with an option is the solution.